### DISCUSSION TITLE
What is the difference between the `ImageBox` control and `Picture` container?

### DISCUSSION THREAD

**Discussion by W98X** [XusinboyBekchanov/VisualFBEditor#513] [Q&A] [answered]
What is the difference between the `ImageBox` control and `Picture` container?

How does somebody decide which one to use?

For example, if I wanted to be able to zoom and pan a large image, which one would be best?

**Comment by XusinboyBekchanov** [comment]
The difference is that Picture is a container.

For these two, the WinAPI class is Static.

**Reply by W98X** [reply]
>The difference is that Picture is a container

What does it mean for it to be a container?

How is a container different to a non-container?

What can be done with a container that can't be done with a non-container?

What are the type of usage scenarios where the picture container would be used instead of the image box?

**Reply by XusinboyBekchanov** [reply, marked as answer]
> What does it mean for it to be a container?
> How is a container different to a non-container?
> What can be done with a container that can't be done with a non-container?

For a non-container, it is not possible to place other controls inside.
If you worked in VB6, there are also ImageBox and Picture controls.

> What are the type of usage scenarios where the picture container would be used instead of the image box?

An ImageBox is just for displaying a picture. And Picture can be used to place other controls, to draw, to display pictures too.

**Reply by W98X** [reply]
OK, that answers my question. Thanks for answering. A Picture container can have other controls inside it, but an ImageBox can't.

**Comment by XusinboyBekchanov** [comment]
> For example, if I wanted to be able to zoom and pan a large image, which one would be best?

For these two, everything will be the same.
